### DISCUSSION TITLE
Uninstall extension?

### DISCUSSION THREAD

**Discussion by Sirmadsen** [mcmonkeyprojects/SwarmUI#448] [General]
How do I properly uninstall/remove a extension? I tried just deleting it from the folder but that caused error messages. I don't see any way to uninstall from the gui.

**Comment by southbayjay** [comment]
> How do I properly uninstall/remove a extension? I tried just deleting it from the folder but that caused error messages. I don't see any way to uninstall from the gui.

That is the correct way but you missed one step, just need to run the update.bat file so swarm can recompile and then start swarm. In your Swarm directory you will see the update-windows.bat file or the one above it if you use Linux or mac. 


![image](https://github.com/user-attachments/assets/8a9f2ef2-ae07-4ae1-8bea-e2519900905c)

**Reply by Sirmadsen** [reply]
Thanks!

**Comment by mcmonkey4eva** [comment]
I was intending to add a button for that a while ago but was busy and forgot -- added now, actual dedicated uninstall button

**Reply by mcmonkey4eva** [reply]
that's just because that's your last loaded workflow, stored in browser session. clear the workflow and it's gone. This is also very unrelated to the original topic
